Rjegl08/switch sample links (#2054)
capire/docs: guides/extensibility/composition.md

5. **Customizing SaaS Solutions** — Customers, who are subscribers of SaaS solutions, can apply the same techniques to adapt SaaS solutions to their needs. They can use prebuilt extension or business data packages, or create their own custom-defined ones.

> ### Examples from [cap/samples](https://github.com/capire)

Author: ```suggestion
### Examples from [sample repositories](https://github.com/capire)
```

> In the following sections, we frequently refer to examples from [cap/samples](https://github.com/capire):

Author: ```suggestion
In the following sections, we frequently refer to examples from the [capire org](https://github.com/capire):
```

> **1)** &nbsp; Get cap/samples:

Author: ```suggestion
**1)** &nbsp; Get capire/bookstore:
```
